src/components/base/BreadCrumb.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div>
+    <span v-for="(link, i) in links" :key="'link-' + link.name">
+      <span v-if="link.path && i !== links.length - 1">
+        <router-link class="plot-nav is-size-5" :to="link.path">
+          <span>{{ link.name }}</span></router-link
+        >
+        <span class="mx-4 is-size-5">/</span>
+      </span>
+      <span v-else class="plot-nav is-size-5">{{ link.name }}</span>
+    </span>
+  </div>
+</template>
+
+<script setup lang="ts">
```

Comment:
I'm really enjoying the syntactic sugar of the setup attribute here and now everything in the script tag is together.